c-pallets/segment-book/src/lib.rs

```diff
@@ -860,7 +875,7 @@ pub mod pallet {
 					);
 				}
 			}
-
+			// why offchain_sign_digest again
```

Comment:
Because I used the same signature in the process of trying, calling multiple trade fairs failed, and one signature must correspond to one transaction, so I made several `offchains_sign_digest()` operation.